### PR TITLE
172763495 testing requests

### PR DESCRIPTION
**Pull request by fadhili-ed**
# testing requests
- validate the user model
- correct violations stated by hound
## blocker
- tests are in this pull request: #5  hence cannot continue until it is merged

### CODE REVIEW

**Comment by fadhili-ed** [comment]
# to do
- I need to validate the user model
# blocker
- problem is, i have an open pull request ( #5 ) that needs to be merged to make this possible.

**Comment by sigu** [comment]
@fadhili-ed the description of the PR still states that you are blocked by #1  which was merged in a while back 🤔

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
```
Comment (sigu) [inline spec/factories/users.rb:3]: Have you dealt with this?

**Comment by fadhili-ed** [comment]
yeah i am trying to find a way to correct that but sipati where i can update description

**Comment by fadhili-ed** [comment]
gotten, i have updated that description now i am going to delete the last one i wrote

**Review by sigu — APPROVED** [review]

**Pull request merged**